thorntail/kubernetes/debug_pod.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
```

Comment:
Thorntail or smallrye because we could provide a microservice with microprofile, jdk because with one class we could test everything in the infrastructure without dependencies